lib/util/progress.js

```diff
@@ -20,46 +20,47 @@
  * THE SOFTWARE.
  *
  */
+const Readline = require('readline');
 
-var modulus = require('../modulus');
+const WIDTH = 20;
+const ANIMATE_TIME = 250;
 
 var Progress = {};
 
-Progress.indeterminate = function(fmt, opts) {
+Progress.Indeterminate = function (fmt, opts) {
```

Comment:
It seems like you uppercased this so that other usage doesn't throw for [`new-cap`](http://eslint.org/docs/rules/new-cap)

`var dbar = new Progress.indeterminate('Restarting [:bar]');`

[eg](https://github.com/onmodulus/modulus-cli/blob/master/lib/controllers/servo.js#L48)

I'm concerned the matching changes not in this PR might result in a forgotten instance. We need to make sure to do a `find` for any remaining instances of `indeterminate` before the final merge.